recipes/nucleoatac/run_test.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# For zlib: https://bioconda.github.io/troubleshooting.html#zlib
+export CFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+export CPATH=${PREFIX}/include
+
```

Comment:
Can I propose a different solution?
Can you remove this test script and use plain `commands` and python module tests:

You export CPATH, you please put in the build.sh script and zlib should be in the run and build environment.